app/assets/stylesheets/about.scss.erb

```diff
@@ -1,10 +1,20 @@
 .about-banner {
   @extend .banner;
-  height: 350px;
-  background: url(<%= image_path 'about/about.jpg' %>);
-  background-size: cover;
-  background-repeat: no-repeat;
-  background-position-y: 40%;
+  height: 500px;
+  /**background: url(<%= image_path 'about/about.jpg' %>);
+  background-size: cover; */
+  background:
+  -webkit-linear-gradient(
+  rgba(255, 255, 255, 0.3),
+  rgba(255, 255, 255, 0.3)
+  ),
+  url(<%= image_path 'about/about.jpg' %>);
```

Comment:
I think it's better to use `image-url('about/about.jpg');` . Then you can remove the .erb ending from the filename.

http://guides.rubyonrails.org/asset_pipeline.html#css-and-sass